src/client/runner/automation/playback/press/utils.js

```diff
@@ -69,3 +70,21 @@ export function getChar (key, shiftModified) {
     return key;
 }
 
+export function getDeepActiveElement (currentDocument) {
+    var doc                   = currentDocument || document;
+    var activeElement         = doc.activeElement && doc.activeElement.tagName ? doc.activeElement : doc.body;
```

Comment:
maybe we should use `isDomElement` instead of `doc.activeElement.tagName` check?